src/cryptonote_core/blockchain.cpp

```diff
@@ -5390,7 +5390,7 @@ void Blockchain::cancel()
 }
 
 #if defined(PER_BLOCK_CHECKPOINT)
-static const char expected_block_hashes_hash[] = "b198c2514bed71ca0a43cba6590090068daf6d207b351d8a4c7d0f0f791c095f";
+static const char expected_block_hashes_hash[] = "ff99bc76e59e0d6268e2d8ed54a2206d828de643fcb13be8d296611fc051a540";
```

Comment:
What is the methodology of learning the proper values of these variables?

Reply:
```
./monero-blockchain-export --output-file checkpoints.dat --block-stop 2430000 --blocksdat
shasum -a256 checkpoints.dat
```

Reply:
I generated the `checkpoints.dat`. What's the next step to learn the hash?

Reply:
`sha256sum checkpoints.dat`

I was using `shasum checkpoints.dat` and was wondering.
OK. Got the same.